autocompleter/registry.py

```diff
@@ -189,53 +190,92 @@ def del_ac_provider_setting(self, ac_name, provider, setting_name):
 registry = AutocompleterRegistry()
 
 
-def add_obj_to_autocompleter(sender, instance, created, **kwargs):
+def add_obj_to_autocompleter(sender, instance, created, add_error_handler = None, remove_error_handler = None, **kwargs):
```

Comment:
is it worth adding tests for this and the remove method? I saw that we use them without going through the signal

Reply:
I think so. I can add some